### PR TITLE
Task50.1-9

### PR DESCRIPTION
**Pull request by Serge-prospect**
@alexsavenok-savenduk please review "Task 50" with 9 sub-tasks included.

### CODE REVIEW

File: Task50/Task50_6/Alerts/Alerts/AlertsTest/JsAlertTests.cs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                Alert.Accept();
+
+                string confirmDemoAcceptText = driver.FindElement(page.ConfirmDemoText).Text;
+
+                Assert.AreEqual(EXPECTED_CONFIRM_ALERT_BOX_TEXT, jsConfirmAlertBoxText, "Confirm alert box text doesn't match.");
+                Assert.AreEqual(EXPECTED_CONFIRM_DEMO_ACCEPT_TEXT, confirmDemoAcceptText, "Accept confirm demo text doesn't match.");
+            }
+            finally { driver.Quit(); }            
+        }
+        [TestMethod]
```
Comment (alexsavenok-savenduk) [inline Task50/Task50_6/Alerts/Alerts/AlertsTest/JsAlertTests.cs:77]: Please, don't forget to split test by empty lines

File: Task50/Task50_7/WaitForUser/WaitForUser/WaitForUser/DynamicDataLoadingDemo.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            _driver = driver;
+            _wait = new WebDriverWait(_driver, TimeSpan.FromSeconds(waitTime));
+        }
+        public static DynamicDataLoadingDemo GoToDynamicDataLoadingDemoPage(IWebDriver driver, int waitTime)
+        {
+            driver.Navigate().GoToUrl(URL);
+            return new DynamicDataLoadingDemo(driver, waitTime);
+        }
+
+        public IWebElement GetelementBy(By elementBy)
```
Comment (alexsavenok-savenduk) [inline Task50/Task50_7/WaitForUser/WaitForUser/WaitForUser/DynamicDataLoadingDemo.cs]: GetElementBy*

**Review by alexsavenok-savenduk — COMMENTED** [review]
I would recommend you don't use Console Application projects to work with tests. Generally, your code looks good.

File: Task50/Task50_6/Alerts/Alerts/AlertsTest/JsAlertTests.cs (hunk 1 above)
Comment (Serge-prospect) [reply Task50/Task50_6/Alerts/Alerts/AlertsTest/JsAlertTests.cs:77]: Accepted that.

File: Task50/Task50_7/WaitForUser/WaitForUser/WaitForUser/DynamicDataLoadingDemo.cs (hunk 2 above)
Comment (Serge-prospect) [reply Task50/Task50_7/WaitForUser/WaitForUser/WaitForUser/DynamicDataLoadingDemo.cs]: Please clarify your comment "GetElementBy*".

Comment (Serge-prospect) [reply Task50/Task50_7/WaitForUser/WaitForUser/WaitForUser/DynamicDataLoadingDemo.cs]: Understood. "Element" should begin from upper case.

**Pull request merged**